workspace/viewer/PdfJsViewer.ts

```diff
@@ -515,49 +522,51 @@ export class PdfJsViewer {
 
   onResize() {
     if (this.currentFile) {
-      this.openCurrentPage();
+      this.showPage(this.currentPage);
     }
   }
 
   pageUp = () => {
     if (this.currentPage > 1) {
       this.currentPage -= 1;
-      this.openCurrentPage();
+      this.showPage(this.currentPage);
     }
   };
 
   pageDown = () => {
     if (this.currentFile && this.currentPage < this.currentFile.numPages) {
       this.currentPage += 1;
-      this.openCurrentPage();
+      this.showPage(this.currentPage);
     }
   };
 
-  getCurrentPage(): number | undefined { return this.currentPage; }
+  getCurrentPage(): number | undefined {
+    return this.currentPage;
+  }
 
   getZoomScale(): number { return this.zoom.current(); }
 
   // Toolbar button handlers
   zoomIn = () => {
-    this.zoom.zoomIn() && this.openCurrentPage();
+    this.zoom.zoomIn() && this.showPage(this.currentPage);
```

Comment:
Please fix zooming. Currently it changes zoom level, but the changes take affect only after subsequent calls to `resetCanvas` and `show`. When integrated into Papeeria it means that one has to recompile the file to change the zooming. I believe that it doesn't work properly in your simple UI either (you wrote that you added simple buttons as far as I remember. By the, please push that simple UI into the repository)

Reply:
Done.